blocks/header/header.js

```diff
@@ -58,12 +58,12 @@ function setupUser(section) {
   const text = section.textContent.trim();
   if (isStored) {
     const user = JSON.parse(isStored);
-    section.innerHTML = `<button id="nav-user-button" class="nav-user-button" data-status="loading">
-        <img src="${user.thumbnailURL}" alt="User Profile Thumbnail"/></span><span>${text}</span>
+    section.innerHTML = `<button id="nav-user-button" class="nav-user-button" aria-labelledby="labelspan" data-status="loading">
+        <img src="${user.thumbnailURL}" alt="User Profile Thumbnail"/></span><span id="labelspan">${text}</span>
```

Comment:
I know the "labelspan" text doesn't appear for users, but could it be more human readable/descriptive for what it is?

Reply:
Sure! Addressed in [0f89981](https://github.com/hlxsites/pga-presidents-cup/pull/3/commits/0f89981c1d8c5b86d5437b48bbd1af01f295c5fb)